js/data.js

```diff
@@ -1,6 +1,8 @@
ame: 'apple sauce', serving_qty: 1, common_type: null, tag_id: '386', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/386_thumb.jpg' }, locale: 'en_US' }, { food_name: 'apple juice', serving_unit: 'cup', tag_name: 'apple juice', serving_qty: 1, common_type: null, tag_id: '385', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/385_thumb.jpg' }, locale: 'en_US' }, { food_name: 'apple cider', serving_unit: 'cup', tag_name: 'apple cider', serving_qty: 1, common_type: null, tag_id: '213', photo: { thumb: 'https://d2eawub7utcl6.cloudfront.net/images/nix-apple-grey.png', highres: null, is_user_uploaded: false }, locale: 'en_US' }, { food_name: 'apple strudel', serving_unit: 'Piece', tag_name: 'apfelstrudel', serving_qty: 1, common_type: null, tag_id: '2190', photo: { thumb: 'https://d2eawub7utcl6.cloudfront.net/images/nix-apple-grey.png', highres: null, is_user_uploaded: false }, locale: 'en_US' }, { food_name: 'apple oatmeal', serving_unit: 'packet', tag_name: 'apple cinnamon oatmeal', serving_qty: 1, common_type: null, tag_id: '3500', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/3500_thumb.jpg' }, locale: 'en_US' }, { food_name: 'applebees shrimp', serving_unit: 'serving', tag_name: "applebee's shrimp", serving_qty: 1, common_type: null, tag_id: '2853', photo: { thumb: 'https://d2eawub7utcl6.cloudfront.net/images/nix-apple-grey.png', highres: null, is_user_uploaded: false }, locale: 'en_US' }, { food_name: 'honeycrisp apple', serving_unit: 'medium (3" dia)', tag_name: 'honeycrisp apple', serving_qty: 1, common_type: null, tag_id: '7827', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/7827_thumb.jpg' }, locale: 'en_US' }, { food_name: 'honeycrisp apples', serving_unit: 'medium (3" dia)', tag_name: 'honeycrisp apple', serving_qty: 1, common_type: null, tag_id: '7827', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/7827_thumb.jpg' }, locale: 'en_US' }, { food_name: 'apple cider vinegar', serving_unit: 'tbsp', tag_name: 'apple cider vinegar', serving_qty: 1, common_type: null, tag_id: '479', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/479_thumb.jpg' }, locale: 'en_US' }, { food_name: 'strawberry applesauce', serving_unit: 'container', tag_name: 'strawberry applesauce', serving_qty: 1, common_type: null, tag_id: '8044', photo: { thumb: 'https://d2eawub7utcl6.cloudfront.net/images/nix-apple-grey.png', highres: null, is_user_uploaded: false }, locale: 'en_US' }, { food_name: 'unsweetened applesauce', serving_unit: 'cup', tag_name: 'unsweetened applesauce', serving_qty: 1, common_type: null, tag_id: '2890', photo: { thumb: 'https://nix-tag-images.s3.amazonaws.com/2890_thumb.jpg' }, locale: 'en_US' }, { food_name: 'evans apples green apples', serving_unit: 'medium apple', tag_name: 'evans apples green apples', serving_qty: 1, common_type: 2, tag_id: '11453', photo: { thumb: 'https://d2eawub7utcl6.cloudfront.net/images/nix-apple-grey.png', highres: null, is_user_uploaded: false }, locale: 'en_US' }],
```

Comment:
Is this meant to be here? If so, it should be on multiple lines, not all jammed on one line. I doubt you did this on purpose, but just a heads up 👍

Reply:
Oops, I forgot to reset this to an empty array before I turned it in. Thanks for the heads up! 👍